docs/docs/genai/eval-monitor/running-evaluation/multi-turn.mdx

```diff
@@ -0,0 +1,171 @@
+import { APILink } from "@site/src/components/APILink";
+
+# Evaluate Conversations
+
+Conversation evaluation enables you to assess entire conversation sessions rather than individual turns. This is essential for evaluating conversational AI systems where quality emerges over multiple interactions, such as user frustration patterns, conversation completeness, or overall dialogue coherence.
+
+:::info[Experimental Feature]
+Multi-turn evaluation is experimental in MLflow 3.7.0. The API and behavior may change in future releases.
+:::
+
+## Overview
+
+Traditional single-turn evaluation assesses each AI response independently. However, many important qualities can only be evaluated by examining the full conversation:
+
+- **User Frustration**: Did the user become frustrated? Was it resolved?
+- **Conversation Completeness**: Were all user questions answered by the end of the conversation?
+- **Dialogue Coherence**: Does the conversation flow naturally?
+
+Multi-turn evaluation addresses these needs by grouping traces into conversation sessions and applying scorers that analyze the entire conversation history.
+
+## Quick Start
+
+Multi-turn evaluation works by grouping traces into conversation sessions using the `mlflow.trace.session` metadata. When building your agent, you can set session IDs on traces to group them into conversations:
+
+```python
+import mlflow
+
+
+@mlflow.trace
+def my_chatbot(question, session_id):
+    mlflow.update_current_trace(metadata={"mlflow.trace.session": session_id})
+    return generate_response(question)
+```
+
+To evaluate conversations, get traces from your experiment and pass them to `mlflow.genai.evaluate`:
+
+```python
+from mlflow.genai.scorers import ConversationCompleteness, UserFrustration
+
+# Get all traces
+traces = mlflow.search_traces(
+    experiment_ids=["<your-experiment-id>"],
+    return_type="list",
+)
+
+# Evaluate all sessions - MLflow automatically groups by session ID
+results = mlflow.genai.evaluate(
+    data=traces,
+    scorers=[
+        ConversationCompleteness(),
+        UserFrustration(),
+    ],
+)
+```
+
+**How it works:** MLflow automatically groups traces by their `mlflow.trace.session` metadata and sorts them chronologically by timestamp within each session. Multi-turn scorers run once per session and analyze the complete conversation history. Multi-turn assessments are logged to the first trace (chronologically) in each session, so you'll see session-level metrics alongside the first interaction in the UI.
```

Comment:
Can we also describe that session-level assessments are displayed in the Sessions tab?